test/README.md

```diff
@@ -253,6 +253,16 @@ There are two scenarios in upgrade tests. One is to install the previous release
 validate whether the Tekton pipeline works. The other is to install the previous release, create the pipelines and tasks,
 upgrade to the current release, and validate whether the Tekton pipeline works.
 
+
+#### Prerequisites for running upgrade tests locally:
+- Change the [PREVIOUS_PIPELINE_VERSION](https://github.com/tektoncd/pipeline/blob/d4c2f75e32a8657f486790da8d665019f5e0550b/test/e2e-tests-upgrade.sh#L28) to the expected previous release version
+- Set up the cluster
+  - Running against a fresh kind cluster
+    - comment out the default [GKE clutser setup](https://github.com/tektoncd/pipeline/blob/d4c2f75e32a8657f486790da8d665019f5e0550b/test/e2e-tests-upgrade.sh#L32)
```

Comment:
it would be nice if this script could be made to work with a kind cluster setup (e.g. via an env var) instead of having people comment out part of it. If that's not feasible to do as part of this PR, I'd suggest creating an issue and linking it here.

Reply:
Thanks Lee! 
Opened issue: https://github.com/tektoncd/pipeline/issues/5689

Reply:
thanks! could you link the issue here in the markdown?